cmd/login/wc_test.go

```diff
@@ -339,10 +439,18 @@ func FakeK8sClient() k8sclient.Interface {
 		if err != nil {
 			panic(err)
 		}
+		err = capz.AddToScheme(scheme)
```

Comment:
Can we just use this scheme instead of maintaining it in two places? https://github.com/giantswarm/kubectl-gs/blob/5118440802d2dfeeb947ba8d41474d2b86166216/pkg/scheme/scheme.go#L37